### DISCUSSION TITLE
Support for Mac media keys

### DISCUSSION THREAD

**Discussion by realbaylen** [MusicPlayerDaemon/MPD#2013] [Ideas]
It would be really cool if, like `mpv`, `mpd` could support the Mac media keys (Previous, Play/Pause,/Skip).

**Comment by owl770** [comment]
It should be possible to set up (for example) the F8 key (play/pause) to execute "mpc toggle", shouldn't it?

**Reply by realbaylen** [reply]
Yes, but in my example of [mpv](https://github.com/mpv-player/mpv), this is not necessary.

**Comment by kingosticks** [comment]
Doesn't mpd-mpris work on MacOS?

**Comment by OliverGuy** [comment]
Necroing to report that [mpd-now-playable](https://git.00dani.me/00dani/mpd-now-playable) works like a charm, and even interfaces nicely with the builtin "now playing" taskbar icon 🎉
